<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Configurator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #0a0a0a;
	color: #ffffff;
}
QLabel {
	color: #ffffff;
}
QLineEdit {
	background-color: #2d2d2d;
	color: #ffffff;
	border: 1px solid #444444;
	padding: 4px;
}
QListWidget {
	background-color: #1e1e1e;
	color: #ffffff;
	border: 1px solid #333333;
}
QDialogButtonBox {
	background-color: #0a0a0a;
	border: none;
}
QPushButton {
	background-color: #2d2d2d;
	color: #ffffff;
	border: 1px solid #444444;
	padding: 5px 12px;
	border-radius: 4px;
}
QPushButton:hover {
	background-color: #3d3d3d;
}
QPushButton:pressed {
	background-color: #4d4d4d;
}
/* Target the OK button specifically when disabled */
QPushButton#okButton:enabled {
	background-color: #555555;  // A grayer tone
	color: #aaaaaa;             // Light grey text
	border: 1px solid #444444;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="porjectNameTitle">
     <property name="font">
      <font>
       <family>Roboto</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="projectNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="dependenciesListTitle">
     <property name="font">
      <font>
       <family>Roboto</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dependencies:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="dependenciesListWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
